adafruit_adxl34x.py

```diff
@@ -394,7 +394,7 @@ def _write_register_byte(self, register, value):
         with self._i2c as i2c:
             i2c.write(self._buffer, start=0, end=2)
 
-    def _config_iterrupt_register(self, register, value):
+    def _config_interrupt_register(self, register, value):
```

Comment:
hey @hartzell you're right, this function isn't used. Can you remove it?

Reply:
yep.